internal/pkg/agent/install/uninstall.go

```diff
@@ -233,19 +234,34 @@ func applyDynamics(ctx context.Context, log *logger.Logger, cfg *config.Config)
 	inputs, ok := transpiler.Lookup(ast, "inputs")
 	if ok {
 		varsArray := make([]*transpiler.Vars, 0)
-		var wg sync.WaitGroup
-		wg.Add(1)
+
+		// Gie some time for the providers to replaces the variables
```

Comment:
"Gie some time for the providers to replaces the variables" -> "Give some time for the providers to replace the variables" typo?